db/populate.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS task;
+
+
+DROP TABLE IF EXISTS user;
+
+
+DROP TABLE IF EXISTS toDo;
+
+
+CREATE TABLE task (id SERIAL PRIMARY KEY,
+                                          module VARCHAR(50) NOT NULL,
+                                                           name VARCHAR(120),
+                                                                   due_date DATE);
+                                                                        
+
+
+CREATE TABLE user (id SERIAL PRIMARY KEY,
+                                          name VARCHAR(100) NOT NULL );
+                                                                    
+
+
+CREATE TABLE to_do (id SERIAL PRIMARY KEY,
```

Comment:
i think the naming should be the same everywhere, i mean we need to decide if the name of the table is toDo or to_to?